### PR TITLE
German strings.xml

### PR DESCRIPTION
**Pull request by peat80**
Translated app strings into german.

### CODE REVIEW

**Comment by peat80** [comment]
#17 Localization

File: src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string name="action_failed">Aktion fehlgeschlagen</string>
+  <string name="add_repository">Paketquelle hinzufügen</string>
+  <string name="address">Addresse</string>
```
Comment (comradekingu) [inline src/main/res/values-de/strings.xml]:
```suggestion
  <string name="address">Adresse</string>
```

File: src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  <string name="all_applications">Alle Apps</string>
+  <string name="all_applications_up_to_date">Keine Updates verfügbar</string>
+  <string name="already_exists">Existiert bereits</string>
+  <string name="always">Immer</string>
+  <string name="anti_features">Unerwünschte Merkmale</string>
+  <string name="application">App</string>
+  <string name="application_name" translatable="false">Foxy Droid</string>
+  <string name="application_not_found">App nicht gefunden</string>
+  <string name="author_email">eMail des Autors</string>
```
Comment (comradekingu) [inline src/main/res/values-de/strings.xml]:
```suggestion
  <string name="author_email">E-Mail des Autors</string>
```

File: src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  <string name="file_format_error_DESC">Ungültiges Dateiformat.</string>
+  <string name="fingerprint">Fingerabdruck</string>
+  <string name="has_advertising">Beinhaltet Werbung</string>
+  <string name="has_non_free_dependencies">Hat unfreie Abhängigkeiten</string>
+  <string name="has_security_vulnerabilities">Hat Sicherheitslücken</string>
+  <string name="http_error_DESC">Serverantwort ungültig.</string>
+  <string name="http_proxy">HTTP-Protokoll</string>
+  <string name="ignore_all_updates">Alle Updates ignorieren</string>
+  <string name="ignore_this_update">Dieses Update ignorieren</string>
+  <string name="incompatible_api_DESC_FORMAT">%1$s (API version %2$d) wird nicht unterstützt. %3$s</string>
```
Comment (comradekingu) [inline src/main/res/values-de/strings.xml]:
```suggestion
  <string name="incompatible_api_DESC_FORMAT">%1$s (API-Version %2$d) wird nicht unterstützt. %3$s</string>
```

File: src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+  <string name="saving_details">Speichere Details</string>
+  <string name="screenshots">Screenshots</string>
+  <string name="search">Suche</string>
+  <string name="select_mirror">Wähle einen Mirror</string>
+  <string name="show_more">Zeige mehr</string>
+  <string name="show_older_versions">Zeige ältere Versionen</string>
+  <string name="signature_FORMAT">Signatur %s</string>
+  <string name="signed_using_unsafe_algorithm">Signiert mit unsicherem Algorithmus</string>
+  <string name="skip">Überspringen</string>
+  <string name="socks_proxy">Socks-Protokoll</string>
```
Comment (comradekingu) [inline src/main/res/values-de/strings.xml]:
```suggestion
  <string name="socks_proxy">SOCKS-Protokoll</string>
```
As per https://de.wikipedia.org/wiki/SOCKS

File: src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+  <string name="proxy_port">Proxy-Port</string>
+  <string name="proxy_type">Proxy-Typ</string>
+  <string name="repositories">Paketquellen</string>
+  <string name="repository">Paketquelle</string>
+  <string name="repository_not_used_DESC">Diese Paketquelle wurde noch nicht verwendet. Bitte Aktivieren um die angebotenen Apps anzusehen.</string>
+  <string name="repository_unsigned_DESC">Unsigniert. App-Liste konnte nicht verifiziert werden. Vorsicht bei Downloads aus unsignierten Paketquellen.</string>
+  <string name="requires_FORMAT">Benötigt %s</string>
+  <string name="save">Speichern</string>
+  <string name="saving_details">Speichere Details</string>
+  <string name="screenshots">Screenshots</string>
```
Comment (comradekingu) [inline src/main/res/values-de/strings.xml:124]:
```suggestion
  <string name="screenshots">Bildschirmfotos</string>
```
As per https://en.wiktionary.org/wiki/Bildschirmfoto

File: src/main/res/values-de/strings.xml [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+  <string name="unverified">Ungeprüft</string>
+  <string name="update">Update</string>
+  <string name="updates">Updates</string>
+  <string name="upstream_source_code_is_not_free">Upstream-Quellcode ist nicht frei</string>
+  <string name="username">Benutzername</string>
+  <string name="username_missing">Benutzername fehlt</string>
+  <string name="validation_index_error_DESC">Index konnte nicht validiert werden.</string>
+  <string name="version_FORMAT">Version %s</string>
+  <string name="versions">Versionen</string>
+  <string name="waiting_to_start_download">Warte auf Download-Start</string>
```
Comment (comradekingu) [inline src/main/res/values-de/strings.xml]:
```suggestion
  <string name="waiting_to_start_download">Warte auf Herunterladen-Start</string>
```

File: src/main/res/values-de/strings.xml [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+  <string name="show_older_versions">Zeige ältere Versionen</string>
+  <string name="signature_FORMAT">Signatur %s</string>
+  <string name="signed_using_unsafe_algorithm">Signiert mit unsicherem Algorithmus</string>
+  <string name="skip">Überspringen</string>
+  <string name="socks_proxy">Socks-Protokoll</string>
+  <string name="sorting_order">Sortierreihenfolge</string>
+  <string name="source_code">Quellcode</string>
+  <string name="source_code_no_longer_available">Quellcode nicht mehr verfügbar</string>
+  <string name="suggested">Empfohlen</string>
+  <string name="sync_repositories">Synchronisiere Paketquellen</string>
```
Comment (comradekingu) [inline src/main/res/values-de/strings.xml:137]:
```suggestion
  <string name="sync_repositories">Synchronisier Paketquellen</string>
```
Why is the ending "e" allowed in imperative form? https://de.wiktionary.org/wiki/synchronisieren

File: src/main/res/values-de/strings.xml (hunk 1 above)
Comment (peat80) [reply src/main/res/values-de/strings.xml]: Correct spelling in german is indeed with one d. 🤔

File: src/main/res/values-de/strings.xml (hunk 6 above)
Comment (peat80) [reply src/main/res/values-de/strings.xml]: Looks and sounds stupid. Would then change this to:

Warte auf Herunterladevorgang

File: src/main/res/values-de/strings.xml (hunk 7 above)
Comment (peat80) [reply src/main/res/values-de/strings.xml:137]: It is indeed allowed and more common with an e at the end. Would rather leave it as is. 😀

File: src/main/res/values-de/strings.xml (hunk 5 above)
Comment (peat80) [reply src/main/res/values-de/strings.xml:124]: The english word is more common in computer world than the german translation i think. Not shure what is better here. 🤔 Could probably changed.

**Pull request closed without merging**